.ci/pytorch/test.sh

```diff
@@ -473,7 +473,7 @@ test_perf_for_dashboard() {
       fi
 
       if [[ "$DASHBOARD_TAG" == *default-true* ]]; then
-        $TASKSET python "benchmarks/dynamo/$suite.py" \
+        TORCHINDUCTOR_ABI_COMPATIBLE=1 TORCHINDUCTOR_CPP_WRAPPER=1 $TASKSET python "benchmarks/dynamo/$suite.py" \
```

Comment:
I think this is probably unintended, but if it's not, should go in as a separate change

Reply:
Yes correct, I just added this to benchmark the change on the perf dashboard. It will not go in, was planning on removing it after removing the WIP.